assets/elements/item/musket/musket.element.yaml

```diff
@@ -4,7 +4,7 @@ builtin: !Musket
   atlas: ./musket.atlas.yaml
 
   max_ammo: 4
-  cooldown_frames: 50
+  cooldown_in_ms: 600
```

Comment:
Could you update this to just be `cooldown` and then use [`humantime_serde`](https://docs.rs/humantime-serde/latest/humantime_serde/) to deserialize it to a duration?

That way we can specify the cooldown as `600ms`.